gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/task/SendEhrExtractRequestHandler.java

```diff
@@ -43,12 +43,12 @@ public boolean prepareAndSendRequest(TransferRequestMessage message) {
             LOGGER.debug(response);
         } catch (WebClientResponseException wcre) {
             LOGGER.error("Received an ERROR response from MHS: [{}]", wcre.getMessage());
-            migrationStatusLogService.addMigrationStatusLog(MigrationStatus.EHR_EXTRACT_REQUEST_ERROR, conversationId, null);
+            migrationStatusLogService.addMigrationStatusLog(MigrationStatus.EHR_EXTRACT_REQUEST_ERROR, conversationId, null, "2");
```

Comment:
"2" is mysterious. I thought GP2GP errors were all 2 digits.

Also, I wonder if it makes sense to refererence an `enum` of valid GP2GP error codes. That way this line of code would become more understandable.

Update: Looking at the NACKReason enum, I can't see a 2. 🤔